### PR TITLE
Add bank icon in header in Plaid flow

### PR DESCRIPTION
**Pull request by puneetlath**
### Details

This PR adds the bank icon on the Plaid bank account selection page when the bank icon is available. When it isn't, we default to the generic bank icon. 

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/166782

### Tests

From an account that has a workspace, but no cards set up: 
1. Click on profile image
2. Click on workspace name to go to workspace settings
3. Click `Expensify Card`
4. Click `Get Started`
4. Click `Log into your bank` 
5. Click `Continue`
5. Choose `Chase`
6. Enter `user_good` and `pass_good` for credentials
7. Click `Continue`
7. Land on account selection page and make sure Chase logo is showing before the text and size/alignment looks right
8. Click back arrow
9. Click `Log into your bank` again
10. userClick `Continue`
10. Choose `Wells Fargo`
11. Enter `user_good` and `pass_good` for credentials
12. Click `Continue`
12. Land on account selection page and make sure default bank icon is showing before the text and size/alignment looks right

### QA Steps

From an account that has a workspace, but no cards set up: 
1. Click on profile image
2. Click on workspace name to go to workspace settings
3. Click `Expensify Card`
4. Click `Get Started`
4. Click `Log into your bank` 
5. Click `Continue`
5. Choose `Chase`
6. Enter `user_good` and `pass_good` for credentials
7. Click `Continue`
7. Land on account selection page and make sure Chase logo is showing before the text and size/alignment looks right
8. Click back arrow
9. Click `Log into your bank` again
10. userClick `Continue`
10. Choose `Wells Fargo`
11. Enter `user_good` and `pass_good` for credentials
12. Click `Continue`
12. Land on account selection page and make sure default bank icon is showing before the text and size/alignment looks right

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Screen Shot 2021-09-30 at 5 59 28 PM
Screen Shot 2021-09-30 at 5 59 28 PM

#### Mobile Web

Screen Shot 2021-09-30 at 5 59 28 PM
Screen Shot 2021-09-30 at 5 59 28 PM

#### Desktop

Screen Shot 2021-09-30 at 5 59 28 PM
Screen Shot 2021-09-30 at 5 59 28 PM

#### iOS

Screen Shot 2021-09-30 at 5 59 28 PM
Screen Shot 2021-09-30 at 5 59 28 PM

#### Android

Screen Shot 2021-09-30 at 5 59 28 PM
Screen Shot 2021-09-30 at 5 59 28 PM

### CODE REVIEW

**Comment by puneetlath** [comment]
cc @shawnborton just double-checking that all the styles look good.

File: src/components/AddPlaidBankAccount.js [hunk 1]
```diff
@@ -177,9 +179,14 @@ class AddPlaidBankAccount extends React.Component {
...
                             <Text style={[styles.mb5]}>{this.props.text}</Text>
                         )}
-                        {/* @TODO there are a bunch of logos to incorporate here to replace this name
-                        https://d2k5nsl2zxldvw.cloudfront.net/images/plaid/bg_plaidLogos_12@2x.png */}
-                        <Text style={[styles.mb5, styles.h1]}>{this.state.institution.name}</Text>
+                        <View style={[styles.flexRow, styles.alignItemsCenter, styles.mb5]}>
+                            <Icon
+                                src={getBankIcon(this.state.institution.name).icon}
+                                height={40}
+                                width={40}
```
Comment (alex-mechler) [inline src/components/AddPlaidBankAccount.js:186]: These hardcoded values should be put in the `variables.js` file.

**Review by alex-mechler — CHANGES_REQUESTED** [review]

File: src/components/AddPlaidBankAccount.js (hunk 1 above)
Comment (puneetlath) [reply src/components/AddPlaidBankAccount.js]: Updated!

**Comment by shawnborton** [comment]
Looks good to me!

**Review by alex-mechler — APPROVED** [review]

**Pull request merged**